posthog/tasks/email.py

```diff
@@ -125,3 +125,22 @@ def send_invite(invite_id: str) -> None:
     )
     message.add_recipient(email=invite.target_email)
     message.send()
+
+
+@app.task(max_retries=1)
+def send_member_join(invitee_uuid: str, organization_id: str) -> None:
+    invitee: User = User.objects.get(uuid=invitee_uuid)
+    organization: Organization = Organization.objects.get(id=organization_id)
+    campaign_key: str = f"member_join_email_org_{organization_id}_user_{invitee_uuid}"
+    message = EmailMessage(
+        campaign_key=campaign_key,
+        subject=f"{invitee.first_name} joined you on PostHog",
+        template_name="member_join",
+        template_context={"invitee": invitee, "organization": organization},
+    )
+    # Don't send this email to the new member themselves
+    members_to_email = tuple(organization.members.exclude(email=invitee.email))
+    if members_to_email:
```

Comment:
I don't think you need either the conversion to `tuple` or the check on whether it's truthy. If the queryset is empty, it will simply not add any recipients in the for loop.